index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>repl.it</title>
```

Comment:
Esse repl.it é de outro código? Hahaha
Se for o caso, cuidado com cópias de outros códigos, pode dificultar o aprendizado de vocês durante o curso ):
Se quiser um template fácil de HTML, o vscode tem um atalho que se você digitar "!" e der ENTER ele gera um template de html pra você (:

Reply:
Oi, Caio! Valeu pelo feedback, ajudou pra caramba!
Não, não é de outro código, acho que esse título é padrão do repl.it e a gente só esqueceu de mudar. Tudo que a gente fez foi se inspirar num layout muito legal sugerido pela Juliana ("https://www.awwwards.com/sites/marco-fyi"), mas não copiamos nada.
A gente pensou em usar o VS CODE, inclusive baixamos uma extensão para trabalhar nele conjuntamente, mas acabou que não funcionou tão bem por vários motivos e fizemos tudo no repl mesmo.